lib/tasks/anonymised.rake

```diff
@@ -0,0 +1,50 @@
+namespace :anonymised do
+  desc "Print an anonymised version of the payload"
+  task :download, [:laa_reference] => :environment do |_, args|
+    submission = Submission.joins(:ordered_submission_versions)
+                           .find_by("application_version.application->>'laa_reference' = ?",
+                                    args[:laa_reference])
+    service_type = submission.application_type == 'crm7' ? :nsm : :prior_authority
+    anonymised = LaaCrimeFormsCommon::Anonymiser.anonymise(
+      service_type,
+      submission.latest_version.application
+    )
+    puts "JSONSTART"
```

Comment:
What's this for? I see you're using it to split later down but I don't understand why you can't just return JSON.

Reply:
So because I'm getting the content by just `> file`, unfortunately the file captures all output of the command, including the lines that get spat out on startup like `I, [2024-12-06T15:29:33.631366 #566]  INFO -- : [PrometheusExporter] Starting server....`. I needed to delineate between that and the actual JSON content.

There's probably a smarter way...

Reply:
Oh right I see.

What if we split out the logic that does the downloading and then for print we just display the data we pulled and for the other task we just add it to the database and do away with the temp file altogether?

Reply:
I'm not sure I follow. I can write tempfile out of the equation by pushing the output from the first command to the input of the second, but I'm still only getting the data from the remote pod to my local machine via stdout, which is "polluted" by all the _other_ stuff that gets written to it by triggering the rake task.